docs/source/traits_user_manual/advanced.rst

```diff
@@ -1157,39 +1157,28 @@ next time its value is requested.
 .. index:: cached_property decorator, depends_on metadata
 
 One strategy to accomplish caching would be to use a private attribute for the
-cached value, and notification listener methods on the attributes that are
+cached value, and notification observer methods on the attributes that are
 depended on. However, to simplify the situation, Property traits support a
-@cached_property decorator and **depends_on** metadata. Use @cached_property to
+@cached_property decorator and **observes** metadata. Use @cached_property to
```

Comment:
It would be nice to make `cached_property` a link here, in passing. (Orthogonal to the PR, I know.)